content/en/docs/reference/glossary/waypoint.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Waypoint
+test: n/a
+---
+
+A waypoint refers to the L7 proxy component of [ambient](/docs/reference/glossary/#ambient) [data plane mode](/docs/reference/glossary/#data-plane-mode).
```

Comment:
L7 is confusing with a  tunnel being L7 based. Suggest to call it simply HTTP: "to the HTTP proxy component"